lib/SILOptimizer/PassManager/PassManager.cpp

```diff
@@ -421,7 +430,19 @@ void SILPassManager::runPassOnFunction(unsigned TransIdx, SILFunction *F) {
   Mod->registerDeleteNotificationHandler(SFT);
   if (breakBeforeRunning(F->getName(), SFT))
     LLVM_BUILTIN_DEBUGTRAP;
+  if (SILForceVerifyAll ||
+      SILForceVerifyAfterPass.end() !=
+          std::find_if(SILForceVerifyAfterPass.begin(),
+                       SILForceVerifyAfterPass.end(), MatchFun)) {
+    forceVerifyAnalyses(F);
+  }
```

Comment:
If you want this to run before and after a pass, it should be called "around-pass" not "after-pass" per convention.